static/usage/v7/select/customization/custom-toggle-icons/index.md

```diff
@@ -0,0 +1,13 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+import react from './react.md';
+import vue from './vue.md';
+import angular from './angular.md';
+
+<Playground
+  version="7"
+  size="350px"
```

Comment:
```suggestion
  size="400px"
```

The "interface popover" playground uses a height of 400px otherwise the popover appears above the button